namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-official-visits-prod/00-namespace.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "hmpps-official-visits-prod"
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "true"
+    cloud-platform.justice.gov.uk/environment-name: "prod"
+    pod-security.kubernetes.io/enforce: restricted
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/slack-channel: "public_move-and-improve"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "move-and-improve-alerts"
+    cloud-platform.justice.gov.uk/application: "HMPPS Official Visits"
```

Comment:
slack-alert-channel = `move-and-improve-alerts-prod` ?   (this is set in other M&I services anyway)

Reply:
I can't see the channel you are suggesting.  I can only see `move-and-improve-alerts`.